pandas/api/indexers/__init__.py

```diff
@@ -1,3 +1,8 @@
-"""Public API for Rolling Window Indexers"""
-from pandas.core.indexers import check_bool_array_indexer  # noqa: F401
-from pandas.core.window.indexers import BaseIndexer  # noqa: F401
+"""
+Public API for Rolling Window Indexers.
+"""
+
+from pandas.core.indexers import check_bool_array_indexer
```

Comment:
cc @jbrockmendel was this function added here recently?

I think `check_bool_array_indexer` is conceptually different from what should belong in this file. This `api.indexers` was intended to be for creating indexers to support `rolling`, `groupby` etc.

Reply:
I think this was a recent PR by @TomAugspurger

Reply:
That was added in https://github.com/pandas-dev/pandas/pull/30308. See the conversation starting around https://github.com/pandas-dev/pandas/pull/30308#issuecomment-570053180

Reply:
Since the method seems ExtensionArray related, thought it could have belonged in `api.extensions`.

Agreed that `api.indexers` isn't the best name. I can put in a PR tonight clarifying (e.g.`api.grouping_indexers` or something)